Converters/Rhino/Speckle.Converters.RhinoShared/ToHost/Raw/BrepXToHostConverter.cs

```diff
@@ -0,0 +1,43 @@
+using Rhino.FileIO;
+using Speckle.Converters.Common;
+using Speckle.Converters.Common.Objects;
+
+namespace Speckle.Converters.Rhino.ToHost.Raw;
+
+public class BrepXToHostConverter : ITypedConverter<SOG.BrepX, List<RG.GeometryBase>>
+{
+  public List<RG.GeometryBase> Convert(SOG.BrepX target) => RawEncodingToHost.Convert(target);
+}
+
+public class SubDXToHostConverter : ITypedConverter<SOG.SubDX, List<RG.GeometryBase>>
```

Comment:
i should have split them into different classes, would be nice if you can, but trivial for now